index.html

```diff
@@ -0,0 +1,178 @@
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4" ><strong>25-29 OCTOBER 2022</strong></h4>
+          <h4 class="head-h4">Virtual Event</h4>
+        </section>
+      </div>
+      <!-- Card section Mobile & Desktop -->
+      <section id="program">
+        <h4 class="text-center">Main Program</h4>
+        <div class="container pb-3">
+          <div class="row justify-content-md-center">
+            <div class="col-md-6 col-lg-3">
+              <div class="card card-program">
+                <img src="./images/lecture-icon.png" class="card-img-top" alt="...">
+                
+                <div class="card-body">
+                  <h5 class="card-title">Lecture</h5>
+                  <p class="card-text">Listen to speakers from various countries about the message of technology and the changing world</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card card-program">
+                <img src="./images/exhibition-icon.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Exhibition</h5>
+                  <p class="card-text">Appreciate various creations applying license of artists organised from Art center</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card card-program">
+                <img src="./images/forum-icon.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Forum</h5>
+                  <p class="card-text">Have the time to share your thoughts and opinions with experts for each topic</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card card-program">
+                <img src="./images/workshop-icon.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Workshop</h5>
+                  <p class="card-text">Try ceating your own work using open source license rather than just watching at it</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card card-program">
+                <img src="./images/network-icon.png" class="card-img-top" alt="...">
+                <div class="card-body">
+                  <h5 class="card-title">Network</h5>
+                  <p class="card-text">Get opportunities to network with tech week affiliates around the world also after the summit</p>
+                </div>
+              </div>
+            </div>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] I think it would be better if you update the styling of the  **Main Program** section to make it stick to the **[Behance given template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**. **e.g.: adding an interaction on cards when one is hovered, all cards to have the expeced background, all cards to be on one row, all cards have the identical width and height as from the template, use the same text color in all the cards**,... Make sure you check all the styling details to stick to the given template design.

| Template Design | Your Design |
| ------------------------- | --------------------------- |
| ![Screenshot from 2022-09-14 22-13-35](https://user-images.githubusercontent.com/83042742/190254890-f66083e5-1803-4798-8664-5bd39a6016b8.png) | ![Screenshot from 2022-09-17 00-10-18](https://user-images.githubusercontent.com/83042742/190812945-6ea42ede-d34a-4eca-aa05-d21237f5f273.png) |


- [ **OPTIONAL** ] **IMPORTANT:** For best semantics syntax, I think it would be better if you use a `<ul>` tag as the container and each in `<li>` tag.